### DISCUSSION TITLE
Difference between Github Roles and Github Permissions

### DISCUSSION THREAD

**Discussion by ktsreddy007** [community/community#128969] [Other Feature Feedback, Questions, & Ideas] [answered]
### GIthub Roles & Permissions

Could someone explain me whats the difference between  Github Roles and Github Permissions as per official documentation its little confusing to understand.

**Comment by Rashidkhan1234567** [comment, marked as answer]
A permission is the ability to perform a specific action. For example, the ability to delete an issue is a permission. A role is a set of permissions you can assign to individuals or teams. Roles work differently for different types of accounts.
